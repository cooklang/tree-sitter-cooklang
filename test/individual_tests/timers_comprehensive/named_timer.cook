~eggs{3%minutes}