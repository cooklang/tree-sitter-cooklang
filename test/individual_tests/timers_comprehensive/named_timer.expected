(recipe
  (step
    (timer
      name: (timer_text)
      (amount
        (quantity)
        (units)))))